src/app/component/SliderWidget/SliderWidget.container.js

```diff
@@ -60,7 +61,7 @@ export class SliderWidgetContainer extends DataContainer {
 
         this.fetchData(
             [SliderQuery.getQuery({ sliderId })],
-            ({ slider }) => this.setState({ slider }),
+            ({ slider }) => this.setState({ slider, isLoaded: true }),
             (e) => showNotification('error', 'Error fetching Slider!', e)
```

Comment:
I believe ESLint would not let this through. Please, **enable** it. The purpose of having it in the project is to prevent repetitive coding style violations people make in their PRs. Namespace comments are lost here.